cluster-autoscaler/cloudprovider/digitalocean/godo/kubernetes.go

```diff
@@ -294,9 +294,10 @@ type KubernetesNodePool struct {
 
 // KubernetesNode represents a Node in a node pool in a Kubernetes cluster.
 type KubernetesNode struct {
-	ID     string                `json:"id,omitempty"`
-	Name   string                `json:"name,omitempty"`
-	Status *KubernetesNodeStatus `json:"status,omitempty"`
+	ID        string                `json:"id,omitempty"`
+	Name      string                `json:"name,omitempty"`
+	Status    *KubernetesNodeStatus `json:"status,omitempty"`
+	DropletID string                `json:"droplet_id,omitempty"`
```

Comment:
Editing this vendored file without applying the change upstream ( https://github.com/digitalocean/godo/blob/7e41cf7a5f534083903fc9238f3e452004f72bae/kubernetes.go#L295-L303 ) will cause drift and make it substantially more challenging to update our godo in the future.  Please open a PR against godo.

Reply:
:+1: https://github.com/digitalocean/godo/pull/255